pkg/templates/node_1.19.go

```diff
@@ -43,6 +43,7 @@ systemd:
           contents: |
             [Service]
             Environment="DOCKER_OPTS=--log-opt max-size=5m --log-opt max-file=5 --ip-masq=false --iptables=false --bridge=none"
+            Environment="DOCKER_CGROUPS=--exec-opt native.cgroupdriver=cgroupfs"
```

Comment:
do we need this for >= 1.19. Shouldn't the default work?

Reply:
Yes, it is needed. Default is now systemd for docker.

Reply:
Or configure the kubelet: https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/#migrating-to-the-systemd-driver